src/components/MostPopular/PopularItem/index.js

```diff
@@ -0,0 +1,16 @@
+import * as S from "./PopularItem.style"
+import Tag from "@/components/Tag"
+
+export default function PopularItem({ item }) {
+  return (
+    <S.Wrapper>
+      <Tag category={item.category} link="http://google.com.br" />
+      <S.Title>{item.title}</S.Title>
+      <S.MetaData>
+        <S.AuthorNDate>
```

Comment:
That's totally ok, but a minor thing: In this case, it would be better to name it something that describes that portion, something like `PostInfo`, `ArticleInfo`, `PostMetada`, etc. UNLESS we know for sure that this will always hold Author and Date. Since we won't be changing this anytime soon it's ok :)